store/store_test.go

```diff
@@ -375,6 +379,40 @@ func TestBlockStoreSaveLoadBlock(t *testing.T) {
 	}
 }
 
+func TestSaveBlockIndexesTxs(t *testing.T) {
+	// Create a state and a block store
+	state, blockStore, cleanup := makeStateAndBlockStore(log.NewTMLogger(new(bytes.Buffer)))
+	defer cleanup()
+
+	// Create a block with 1000 txs
```

Comment:
[question] does the following code make a block with 1000 txs or 1000 blocks? Since each loop execution calls `makeBlock`, I think the latter. If so:

```suggestion
	// Create 1000 blocks
```

It looks like `makeTxs` always creates 10 transactions so pretty sure the comment is incorrect

Reply:
it's supposed to be 1000 blocks, txs was by mistake